### ISSUE TITLE
FastText and StarSpace support

### ISSUE THREAD

**Issue by loretoparisi** [lutzroeder/netron#152]
Thanks for this amazing model visualizer! Please add support for the FastText and StartSpace models.

**Comment by lutzroeder** [comment]
Hi @loretoparisi, can you share the model files?

**Comment by loretoparisi** [comment]
@lutzroeder yes, you  can easily train a simple Word2Vec model (plus last softmax layer for classification in this case) here in [FastText.js](https://github.com/loretoparisi/fasttext.js) running the example for sms spam/ham or you can get other example pre-trained models here https://fasttext.cc/docs/en/supervised-models.html

**Comment by lutzroeder** [comment]
What format and data do these files contain? Looks like they are binary dumps without a model graph?
